gazebo_ros/include/gazebo_ros/executor.hpp

```diff
@@ -1,4 +1,4 @@
-// Copyright 2018 Open Source Robotics Foundation, Inc.
+// Copyright 2014 Open Source Robotics Foundation, Inc.
```

Comment:
?

Reply:
Since this directly copied from [`rclcpp::MultiThreadedExecutor`](https://github.com/ros2/rclcpp/blob/adebda52c541533d878a4d0bae2ed112403c1e2a/rclcpp/include/rclcpp/executors/multi_threaded_executor.hpp), I kept the same copyright notice.